### ISSUE TITLE
Sweep Does Not Run via API

### ISSUE THREAD

**Issue by Monkeymerlot** [levylabpitt/Multichannel-Lockin#70]
![image](https://user-images.githubusercontent.com/37567672/193652843-55b3dfaf-d689-4ead-8a60-afc88622a49e.png)

If I press the play button on the bottom, nothing happens. I do not get any updates of graphs, and in addition the timer does not start.

The DAQ status looks fine:
![image](https://user-images.githubusercontent.com/37567672/193653050-688d75c2-b637-426e-9453-da626fefbaf6.png)

**Comment by Monkeymerlot** [comment]
Potentially related to issue #69

**Comment by Monkeymerlot** [comment]
OK so I looked back into this today. I think that the issue is not necessarily with the sweep in general, but rather that it seems to get broken after using the API. 

What I mean by this is that I tried to use the sweep control from the source in LabVIEW (UI) and it worked fine. Then I tried the compiled version (v2.15.3.17) (UI) and it worked as well. Then I tried using the API and it doesn't work. Furthermore, it won't work in the UI anymore either after trying with the API.

It will wait the time before sweeping if you have that selected (both in the UI and API) but then won't do the sweep.
